ffi/SDL2_0.lua

```diff
@@ -423,7 +424,7 @@ function S.waitForEvent(sec, usec)
         end
         local x = is_finger and event.tfinger.x * S.w or event.button.x
         local y = is_finger and event.tfinger.y * S.h or event.button.y
-        setPointerDownState(slot, true)
+        setPointerDownState(slot, true, is_finger)
```

Comment:
Oh right, in the specific sway bug scenario this one will only trigger once. But in that case this would need a lot of comments to explain what it's doing since this won't solve the generic scenario that a naive reader would expect to be solved.

Reply:
Yeah, more comments would be good; because that MOUSEMOTION event in the middle of the sandwich is hella weird ;o).

(I wouldn't mind a link to https://github.com/koreader/koreader/issues/10417#issuecomment-1542781926 in said comment, as this clearly showcases the issue).